pvnet/app.py

```diff
@@ -313,19 +351,27 @@ def app(
             elevation = elevation[:, -preds.shape[1] :]
             sun_down_mask = elevation < MIN_DAY_ELEVATION
 
-            # Zero out after sundown
-            preds[sun_down_mask] = 0
```

Comment:
why remove this?

Reply:
ah i see now, for the below to do the sun masking. 
Q: do we need to sun mask on national if we have done it on gsp?

Reply:
Yeh I think we do. The summation model will occassionally predict positive values during the night, mainly for the 90th percentile line. So it does need to be masked. Also the summation model was trained on non-sun-masked GSP inputs. To keep things the same I don't mask the GSP inputs fed into the summation model here either